lib/wibox/widget/graph.lua

```diff
@@ -621,25 +626,27 @@ function graph:draw_values(cr, width, height, drawn_values_num)
     -- pass data pertaining to the same drawing session between callbacks, but
     -- all underscore-prefixed keys are reserved for future use by the widget.
```

Comment:
If you're going to drop the leading underscores convention, this doc line is probably obsolete.